base_export_manager/models/ir_exports_line.py

```diff
@@ -53,104 +62,106 @@ def _default_model1_id(self):
+        if name != self.name:
+            self.name = name
 
-    @api.multi
+    @api.one
     @api.depends("field1_id")
     def _compute_model2_id(self):
         """Get the related model for the second field."""
         ir_model = self.env["ir.model"]
-        for s in self:
-            s.model2_id = (
-                s.field1_id.ttype and
-                "2" in s.field1_id.ttype and
-                ir_model.search([("model", "=", s.field1_id.relation)]))
+        self.model2_id = (
+            self.field1_id.ttype and
+            "2" in self.field1_id.ttype and
+            ir_model.search([("model", "=", self.field1_id.relation)]))
 
-    @api.multi
+    @api.one
     @api.depends("field2_id")
     def _compute_model3_id(self):
         """Get the related model for the third field."""
         ir_model = self.env["ir.model"]
-        for s in self:
-            s.model3_id = (
-                s.field2_id.ttype and
-                "2" in s.field2_id.ttype and
-                ir_model.search([("model", "=", s.field2_id.relation)]))
+        self.model3_id = (
+            self.field2_id.ttype and
+            "2" in self.field2_id.ttype and
+            ir_model.search([("model", "=", self.field2_id.relation)]))
 
-    @api.multi
+    @api.one
+    @api.depends("field3_id")
+    def _compute_model4_id(self):
+        """Get the related model for the third field."""
+        ir_model = self.env["ir.model"]
+        self.model4_id = (
+            self.field3_id.ttype and
+            "2" in self.field3_id.ttype and
+            ir_model.search([("model", "=", self.field3_id.relation)]))
+
+    @api.one
     @api.depends('name')
     def _compute_label(self):
         """Column label in a user-friendly format and language."""
-        translations = self.env["ir.translation"]
-        for s in self:
+        try:
             parts = list()
-            for num in range(1, 4):
-                field = s.field_n(num)
+            for num in range(1, 5):
+                field = self.field_n(num)
                 if not field:
                     break
-
                 # Translate label if possible
                 parts.append(
-                    translations.search([
-                        ("type", "=", "field"),
-                        ("lang", "=", self.env.context.get("lang")),
-                        ("name", "=", "%s,%s" % (s.model_n(num).model,
-                                                 field.name)),
-                    ]).value or
-                    field.display_name)
-            s.label = ("%s (%s)" % ("/".join(parts), s.name)
-                       if parts and s.name else False)
+                    self.env[self.model_n(num).model]._fields[field.name]
+                    .get_description(self.env)["string"])
+            self.label = ("%s (%s)" % ("/".join(parts), self.name)
+                          if parts and self.name else False)
+        except KeyError:
+            pass
```

Comment:
IMO it would also be a little nicer if we guarded a little closer to the key error. We could then just sub the `pass` for a `return` to accomplish the same effect